nwb_conversion_tools/datainterfaces/behavior/movie/moviedatainterface.py

```diff
@@ -129,12 +121,10 @@ def run_conversion(
             If the processing module specified by module_name does not exist, it will be created with this description.
             The default description is the same as used by the conversion_tools.get_module function.
         """
+        from .movie_utils import VideoCaptureContext
```

Comment:
Is there a reason to do this import local to the function? I think it would be fine to invoke up at the top

Reply:
yeah it can move up, https://github.com/catalystneuro/nwb-conversion-tools/pull/372/commits/8b8ca554b247cd2ff537a2361e47dde9001c1188